docker/provision/vnc.mk

```
@@ -0,0 +1,39 @@
+VNC_DISPLAY	!=	echo '${VNC_HOST_PORT}' | awk '{ sub(/590?/, "", $$0); print $$0 }'
+NOVNC_PORT	=	680${VNC_DISPLAY}
+
+vnc-setup: turbovnc virtualgl novnc vnc-window-manager vnc-script vnc-entrypoint
+
+turbovnc:
+	wget -nv -O ./turbovnc.deb "https://downloads.sourceforge.net/project/turbovnc/2.2.90%20%283.0%20beta1%29/turbovnc_2.2.90_amd64.deb?ts=gAAAAABiCTM5OO6vxolPny2srVISQf6mZ-9O1vTxAGZsOCferRYGKwVMmrd8fY7t5QV787Hzmrq5DXn9Xu4kHzC9VyyEgRtJ0g%3D%3D&use_mirror=autoselect&r=https%3A%2F%2Fsourceforge.net%2Fprojects%2Fturbovnc%2Ffiles%2F2.2.90%2520%25283.0%2520beta1%2529%2Fturbovnc_2.2.90_amd64.deb%2Fdownload"
```

Ovi tokeni mozda isticu: 
```suggestion
	wget -nv -O ./turbovnc.deb "https://phoenixnap.dl.sourceforge.net/project/turbovnc/2.2.90%20%283.0%20beta1%29/turbovnc_2.2.90_amd64.deb"
```

---

Problem sa direktnim linkovanjem mirrora je sto brzina preuzimanja zna da varira drasticno. Jednom moze biti reda 10Mbps, a drugi put ne prelazi ni 100Kbps. Prvobitno sam stavio link ka mirroru, ali sam zbog navedenog problema stavio da njihv load balancer odredi.